src/main/kotlin/br/unipar/plano/interfaces/rest/solicitacaoprocedimento/SolicitacaoProcedimentoResource.kt

```diff
@@ -0,0 +1,35 @@
+package br.unipar.plano.interfaces.rest.solicitacaoprocedimento
+
+import br.unipar.plano.domain.solicitacaoprocedimento.model.IdSolicitacaoProcedimento
+import br.unipar.plano.domain.solicitacaoprocedimento.service.SolicitacaoProcedimentoService
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("/solicitacao-liberacao-procedimento")
```

Comment:
Você pode agregar tudo isso diretamente no recurso em que está operando.
```suggestion
@RequestMapping("/procedimento")
```